### PR TITLE
Add primepi function

### PR DESCRIPTION
**Pull request by rikardn**
Symbolic prime counting function

### CODE REVIEW

**Comment by rikardn** [comment]
Sorry for many attempts to get the checks to pass. One future idea is to optionally use `libprimecount` for this. It seems to be the fastest available and it is BSD-licensed

**Review by lkeegan — APPROVED** [review]
lgtm!

One question, should this function be parsed by the infix parser? If so it should be added to the list of functions here: https://github.com/symengine/symengine/blob/master/symengine/parser/parser.cpp#L55

**Comment by rikardn** [comment]
Thanks @lkeegan. I added primepi to the parser.

File: symengine/ntheory_funcs.h [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+#include <symengine/basic.h>
+#include <symengine/symengine_casts.h>
+#include <symengine/constants.h>
+#include <symengine/functions.h>
+
+namespace SymEngine
+{
+
+class PrimePi : public OneArgFunction
```
Comment (isuruf) [inline symengine/ntheory_funcs.h:12]: Should this just be moved to `functions.h`?

Comment (rikardn) [reply symengine/ntheory_funcs.h:12]: Good point. I can do it in a future PR.

**Pull request merged**